explore/src/main/webapp/main.jsx

```diff
@@ -29,15 +29,35 @@ if (import.meta.env.DEV) {
   process.env = { CATS_EFFECT_TRACING_MODE: 'none' };
 }
 
-Explore.runIOApp();
+// Get dynamic enums static data from ODB rest point and into a global variable.
+// ODB rest URL is resolved from environments.conf.json.
+fetch('/environments.conf.json').then((response) => {
```

Comment:
This file is available at build-time, right? We should be able to do `import environment from "/environments.conf.json"`, I think (or the similar in scala code), and Vite will resolve it, saving us a network hop.

Reply:
Good idea! That should work, I'll implement it.